bin/lib/installation.py

```diff
@@ -416,13 +420,17 @@ def build(self, buildfor):
         if self.build_config.build_type == "":
             raise RuntimeError('No build_type')
 
-        sourcefolder = os.path.join(self.install_context.destination, self.install_path)
-        builder = LibraryBuilder(logger, self.language, self.context[-1], self.target_name, sourcefolder,
-                                 self.install_context, self.build_config)
-
-        if self.build_config.build_type == "cmake":
-            return builder.makebuild(buildfor)
-        elif self.build_config.build_type == "make":
+        if self.build_config.build_type in ["cmake", "make"]:
```

Comment:
nice use of `in` 👍🏻 

(wouldn't it be lovely if python supported `switch` eh? coming soon in python 3.10 is `match`...but one thing at a time)